packages/Alert/src/hooks/useBrand.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react";
+
+export const useBrand = (): boolean => {
+    const [isWorkleap, setIsWorkleap] = useState(false);
+
+    useEffect(() => {
+        const observer = new MutationObserver(mutations => {
+            mutations.forEach(mutation => {
+                if (mutation.type === "attributes" && mutation.attributeName === "data-brand") {
+                    const brand = document.documentElement.getAttribute("data-brand");
+                    setIsWorkleap(brand === "workleap");
+                }
+            });
+        });
+
+        observer.observe(document.documentElement, { attributes: true });
+
+        return () => {
+            observer.disconnect();
+        };
+    }, []);
+
+    return isWorkleap;
+};
```

Comment:
everywhere else in the codebase, the way it's done is with : 
```
    const getBrand = (): string => {
        return document.documentElement.getAttribute("data-brand") ?? "igloo";
    };
```

why is this one different?

Reply:
This is iffy and might not always work. The data-attribute is set by storybook when the global variable is updated. For a basic switch, it might work, but this hook I created watches the actual data-attribute. I feel more comfortable with this, especially since we're not testing with OV here.